streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -52,6 +54,8 @@
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.kafka.streams.processor.internals.ProcessorContextImpl.KEY_SERIALIZER;
+import static org.apache.kafka.streams.processor.internals.ProcessorContextImpl.VALUE_SERIALIZER;
```

Comment:
prop: I think we should move those to `InternalProcessorContext`. IMO, it would be cleaner for a mock not to have a direct dependency to the class it mocks. The new consolidated mock for the internal processor context will not extend `AbstractProcessorContext` but only `MockProcessorContext`. Thus, the common ancestor will be `InternalProcessorContext`.
Sorry for bothering you because of those constants after that @vvcephei has already bothered you. :-)

Reply:
Ack